### PR TITLE
Setup files for match statistics

### PR DESCRIPTION
**Pull request by BaoKhanh99**
Related Tickets
• [#47661](https://edu-redmine.sun-asterisk.vn/issues/47661)
What

- Setup files for match statistics
How

- create circle chart by javascipt in views side
- update seed file
- update data type goal result

Screenshot from 2022-03-28 08-34-02
Screenshot from 2022-03-28 08-47-24

### CODE REVIEW

**Comment by BaoKhanh99** [comment]
ready

**Comment by binhpt-1239** [comment]
ok

File: app/controllers/admin/soccer_matches_controller.rb [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-  before_action :load_soccer_match, only: %i(edit update)
-  before_action :load_teams_and_tournament, only: %i(new edit)
+  before_action :load_soccer_match, only: %i(edit update show)
+  before_action :load_teams_and_tournaments, only: %i(new edit)
 
   def index
     @pagy, @soccer_matches = pagy SoccerMatch.includes(:home_team, :guest_team)
                                              .newest
+    @win = Currency.where(currency_type_id: 3).sum(:amount)
+    @lose = Currency.where(currency_type_id: 4).sum(:amount)
```
Comment (vanvtt-0952) [inline app/controllers/admin/soccer_matches_controller.rb:9]: `.where(currency_type_id: 3)` tận dụng scope đi em nhé

File: app/javascript/packs/chart.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    labels: xValues,
+    datasets: [{
+      backgroundColor: barColors,
+      data: yValues
+    }]
+  },
+  options: {
+    title: {
+      display: true,
+      text: "Thống kê thu nhập của tất cả trận đấu"
```
Comment (vanvtt-0952) [inline app/javascript/packs/chart.js:23]: I18n nhé

dùng gem I18n-js nè

File: app/views/admin/soccer_matches/show.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+show Match Statistics here
```
Comment (vanvtt-0952) [inline app/views/admin/soccer_matches/show.html.erb:1]: ko có dùng đến thì ngại j ko xóa thanh niên

**Pull request merged**